<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogScheduleProfessor</class>
 <widget class="QDialog" name="dialogScheduleProfessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schedule</string>
  </property>
  <widget class="QGroupBox" name="groupBoxEditDataSchedule">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1181</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit data</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Teach date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Period:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>200</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start date semester:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>260</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>End date semester:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Classroom code:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateScheduleTeachDate">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEditScheduleTimeStart">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>80</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="displayFormat">
     <string>hh:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsbSchedulePeriod">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>140</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>140</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>hours</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditScheduleStartDate">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>200</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditScheduleEndDate">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>260</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cbScheduleClassroomCode">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>320</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSchedule">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxRb">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Radio buttons</string>
    </property>
    <widget class="QRadioButton" name="rbAddSchedule">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Add schedule</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbUpdateSchedule">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Update schedule</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbDeleteSchedule">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete schedule</string>
     </property>
    </widget>
   </widget>
   <widget class="QTreeView" name="treeViewCourseAvailable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>521</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select an assigned professor to edit schedule. (Click on given rows)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxScheduleContent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>1181</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Content</string>
   </property>
   <widget class="QTableView" name="tvSchedule">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>1141</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
